nipy/algorithms/registration/groupwise_registration.py

```diff
@@ -18,20 +18,18 @@
 import warnings
 
 import numpy as np
-
-from ...externals.six import string_types
-
 from nibabel.affines import apply_affine
 
+from ...externals.six import string_types
```

Comment:
This import should go at the top, because it's a sort-of special import, like a system import.